controls/cis_eks.yml

```diff
@@ -0,0 +1,647 @@
+      levels:
+      - level_1
+      rules: []
+  - id: '3.2'
+    title: >-
+      3.2 Kubelet
+    description: |-
+      This section contains recommendations for kubelet configuration.
+
+      Kubelet settings may be configured using arguments on the running kubelet executable, or
+      they may be taken from a Kubelet config file. If both are specified, the executable argument
+      takes precedence.
+
+      To find the Kubelet config file, run the following command:
+
+          ps -ef | grep kubelet | grep config
+
+      If the --config argument is present, this gives the location of the Kubelet config file. This
+      config file could be in JSON or YAML format depending on your distribution.
+    status: pending
+    levels:
+    - level_2
+    controls:
+    - id: 3.2.1
+      title: >-
+        3.2.1 Ensure that the --anonymous-auth argument is set to false (Automated)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.2
+      title: >-
+        3.2.2 Ensure that the --authorization-mode argument is not set to AlwaysAllow (Automated)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.3
+      title: >-
+        3.2.3 Ensure that the --client-ca-file argument is set as appropriate (Manual)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.4
+      title: >-
+        3.2.4 Ensure that the --read-only-port is secured (Manual)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.5
+      title: >-
+        3.2.5 Ensure that the --streaming-connection-idle-timeout argument is not set to 0 (Manual)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.6
+      title: >-
+        3.2.6 Ensure that the --protect-kernel-defaults argument is set to true (Automated)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.7
+      title: >-
+        3.2.7 Ensure that the --make-iptables-util-chains argument is set to true (Automated)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.8
+      title: >-
+        3.2.8 Ensure that the --hostname-override argument is not set (Manual)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+    - id: 3.2.9
+      title: >-
+        3.2.9 Ensure that the --eventRecordQPS argument is set to 0 or a level which ensures appropriate event capture (Automated)
+      status: pending
+      levels:
+      - level_2
+      rules: []
+    - id: 3.2.10
+      title: >-
+        3.2.10 Ensure that the --rotate-certificates argument is not set to false (Manual)
+      status: pending
+      levels:
+      - level_2
+      rules: []
+    - id: 3.2.11
+      title: >-
+        3.2.11 Ensure that the RotateKubeletServerCertificate argument is set to true (Manual)
+      status: pending
+      levels:
+      - level_1
+      rules: []
+- id: '4'
```

Comment:
Do we need to fill in the section description like we do for other sections?

Reply:
We could. However, to not make this PR bigger than it already is, I suggest we do so as part of evaluating section 4.

Reply:
Just wanted to make sure. Thanks for the update, let's do it in a follow up.